clang/test/Driver/sycl-intelfpga-aoco-emu.cpp

```diff
@@ -0,0 +1,94 @@
+// UNSUPPORTED: system-windows
+
+/// -fintelfpga static lib (aoco)
```

Comment:
If this flow is similar to what we have for non-FPGA-HW targets when treating static archives, adding the first check block could be sufficient (perhaps within `sycl-intelfpga-aoco.cpp`, although I understand that the archive would need to be re-compiled). Provided that existing tests provide sufficient coverage for the "regular" flow, of course.

Reply:
Yes, this flow should be the same as other static archives.  Using sycl-intelfpga-aoco.cpp should be fine without re-compiling the archive - it's just a matter of what command line option is used.